### PR TITLE
trigger_to_ignore

### PR DESCRIPTION
**Pull request by robro-ayush**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,15 @@ Libraries to save images.
 Can add multiple options of camera name, application name, set name, category name, file name numbering... 
 
 Can add options to take from live image, or for specific cameras
+
```
Comment (UmangRobro) [inline README.md]: Please remove this unnecessary comment.

File: README.md [hunk 2]
```diff
@@ -3,3 +3,15 @@ Libraries to save images.
...
+
+### How to use dataCollection.launch
+
+1. Clone https://github.com/hs29590/robro_cloud
+2. Install packages as mentioned in Readme.
+3. install robroCloud as mentioned in Readme.
```
Comment (UmangRobro) [inline README.md]: Where or which readme

**Review by UmangRobro — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (robro-ayush) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (robro-ayush) [reply README.md]: done

**Review by UmangRobro — APPROVED** [review]

**Pull request merged**